temporal-server/authorizer/claimer_test.go

```diff
@@ -0,0 +1,207 @@
+package authorizer_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/canonical/charmed-temporal-image/temporal-server/authorizer"
+	mock "github.com/canonical/charmed-temporal-image/temporal-server/authorizer/mocks"
+	"github.com/golang/mock/gomock"
```

Comment:
If this is a new project, it might be a good idea to instead use https://github.com/uber-go/mock.

golang/gomock was deprecated in favor of it, see: https://github.com/golang/mock

The API is still the same IIRC, it's just a matter of picking the future-proof thing.